src/extend/diastereotopic/getGroupedDiastereotopicAtomIDs.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
+/**
+ * This function groups the diasterotopic atomIds of the molecule based of equivalence of atoms. This
+ * equivalente can be at the level of chemical equivalente(Default) or at the level of magentic equivalence of protons
+ */
```

Comment:
Please add jsdocs for possible options
* {object} [opitons={}]
* {xxxx} [optoins.yyy=zzz]

Reply:
Is it the expected format?

* {object}[options={}]
 * {string}[options.atomLabel] Return only the atoms of the given atomLabel. By default it returns all the explicit atoms in the molecule

Reply:
Yes it is the way we document options. If there is a default value we put [options.atomLabel="C"]